### PR TITLE
Update two example files to API version 2.9

### PR DESCRIPTION
**Pull request by kevinqz**
Update example files to API version 2.9:
- simple_create.py
- ad_creation_utils.py

### CODE REVIEW

**Comment by kevinqz** [comment]
Ping on this.

**Comment by kevinqz** [comment]
Guys, I updated the files considering the new version of the API, is there anything I can improve for the Pull Request to be accepted?

**Comment by advance512** [comment]
Why is this not merged?

File: examples/ad_creation_utils.py [hunk 1]
```diff
@@ -18,17 +18,18 @@
...
 
-from facebookads.objects import (
-    AdCampaign,
-    AdSet,
-    AdGroup,
-    AdImage,
-    AdCreative,
-    TargetingSpecsField,
-)
+from facebookads.adobjects.adaccount import AdAccount
```
Comment (daphyFB) [inline examples/ad_creation_utils.py]: AdAccount is not used anywhere else in the file.

File: examples/ad_creation_utils.py [hunk 2]
```diff
@@ -18,17 +18,18 @@
...
+from facebookads.adobjects.campaign import Campaign
+from facebookads.adobjects.adset import AdSet
+from facebookads.adobjects.targeting import Targeting
+from facebookads.adobjects.ad import Ad
+from facebookads.adobjects.adimage import AdImage
+from facebookads.adobjects.adcreative import AdCreative
+from facebookads.adobjects.targetinggeolocation import TargetingGeoLocation
```
Comment (daphyFB) [inline examples/ad_creation_utils.py]: ditto. This is not used anywhere else in the file.

File: examples/simple_create.py [hunk 3]
```diff
@@ -74,26 +74,30 @@
...
+        campaign=config['campaign_id'],
+        status=AdSet.Status.paused,  # Default is False but let's keep this test ad paused
     )
     print('**** Done!')
 
     # Get the preview and write an html file
-    preview = my_ad.get_ad_preview(params={
-        AdPreview.Field.ad_format: AdPreview.AdFormat.right_column_standard
+    preview = my_ad.get_previews(params={
+        'ad_format':  AdPreview.AdFormat.right_column_standard,
```
Comment (daphyFB) [inline examples/simple_create.py]: two spaces after ':'

File: examples/simple_create.py [hunk 4]
```diff
@@ -74,26 +74,30 @@
...
+        billing_event=AdSet.BillingEvent.impressions,
+        bid_amount=53,  # $0.53 / thousand
         daily_budget=1000,  # $10.00 per day
 
         age_min=13,
         age_max=65,
 
-        paused=True,  # Default is False but let's keep this test ad paused
+        campaign=config['campaign_id'],
+        status=AdSet.Status.paused,  # Default is False but let's keep this test ad paused
```
Comment (daphyFB) [inline examples/simple_create.py]: line too long; prefer 80 characters

**Comment by kevinqz** [comment]
Thanks for the update, @daphyFB! I've just updated the codes in a new commit.

**Pull request merged**